### PR TITLE
Update AWS Customer-Managed VPC readme instructions

### PR DESCRIPTION
**Pull request by frenchfrywpepper**
In the process of updating the Redpanda public docs for this feature which will involve referencing this readme for some steps.

### CODE REVIEW

File: customer-managed/aws/README.md [hunk 1]
```diff
@@ -1,268 +1,115 @@
...
-AWS project. (Note that the link to the minimum permissions is provided simply as documentation, we do not anticipate 
-that you would create the role in that file, it is provided only as a way to communicate the actions that will be 
-required of the employee running rpk byoc apply.)
-
-# Create VPC and other sensitive resources
-
-The [terraform code](https://github.com/redpanda-data/cloud-examples/customer-managed/aws/terraform) contains the 
-specifications for what resources are expected to exist and how they are expected to be configured.
+This repository contains [Terraform](https://developer.hashicorp.com/terraform) code that describes the resources
+customers are responsible for creating in association with a Redpanda Customer-Managed VPC cluster. These resources
```
Comment (micheleRP) [inline customer-managed/aws/README.md:4]:
```suggestion
customers are responsible for creating in association with a Redpanda customer-managed VPC cluster. These resources
```

File: customer-managed/aws/README.md [hunk 2]
```diff
@@ -1,268 +1,115 @@
...
-* VPC Endpoint Route Table Associations
-* Security Groups
-* Security Group Rules
-* S3 Buckets
-* S3 Bucket Server-Side Encryption Configurations
-* S3 Bucket Versioning
-* S3 Bucket Ownership Controls
-* DynamoDB Table
+TODO: doc link here (would be nice to have an overview doc of Customer-Managed VPC that is agnostic of cloud provider)
+For more information on Redpanda's Customer-Managed VPC offering see...
```
Comment (micheleRP) [inline customer-managed/aws/README.md:8]:
```suggestion
For more information on Redpanda's customer-managed VPC offering, see...
```
no link for this today, but you could link to this BYOC architecture? https://docs.redpanda.com/redpanda-cloud/get-started/cloud-overview/#byoc-architecture

File: customer-managed/aws/README.md [hunk 3]
```diff
@@ -1,268 +1,115 @@
...
 
-Some of these resources may already exist in your AWS project in which case you can remove that item from the terraform
-and modify references to use a data block instead of the resource.
+> There may be resources in this repository that already exist within your environment that you wish to skip creation
+of (e.g. the VPC). Variables are provided for this purpose, more details on the variables later in this document.
```
Comment (micheleRP) [inline customer-managed/aws/README.md:11]:
```suggestion
> There may be resources in this repository that already exist within your environment (for example, the VPC) that you don't want to create. Variables are provided for this purpose.
```

File: customer-managed/aws/README.md [hunk 4]
```diff
@@ -1,268 +1,115 @@
...
-In particular the resources in the 
-[iam_rpk_user.tf](https://github.com/redpanda-data/cloud-examples/customer-managed/aws/terraform/iam_rpk_user.tf) file 
-are not intended to be created but are only provided to serve as documentation of the minimum viable permissions for the 
-employee who will run rpk byoc apply.
+# Prerequisites
 
-This terraform module outputs a number of ARN values, these will be used as inputs for the Public API calls.
+1. Access to an AWS project in which you would like to create your cluster.
+2. Knowledge of your internal VPC and subnet configuration (e.g. the ARN of the VPC and private subnets)
+3. Permission to create, modify and delete the resources described by this Terraform
```
Comment (micheleRP) [inline customer-managed/aws/README.md:17]:
```suggestion
1. Access to an AWS project where you want to create your cluster
2. Knowledge of your internal VPC and subnet configuration (for example, the ARN of the VPC and private subnets)
3. Permission to create, modify, and delete the resources described by this Terraform
```

File: customer-managed/aws/README.md [hunk 5]
```diff
@@ -1,268 +1,115 @@
...
+4. [Terraform](https://developer.hashicorp.com/terraform/install) version 1.8.5 or later
 
-# Create Redpanda Cloud Network using Public API
+# Setup
 
-After [authenticating](https://redpanda-api-documentation.netlify.app/#auth) to the Public API, issue a create network 
-request using the outputs from the terraform in the previous step.
+> You may wish to configure [remote state](https://developer.hashicorp.com/terraform/language/state/remote) for this 
+> Terraform. For simplicity these instructions will assume local state.
```
Comment (micheleRP) [inline customer-managed/aws/README.md:23]:
```suggestion
> You may want to configure [remote state](https://developer.hashicorp.com/terraform/language/state/remote) for this 
> Terraform. For simplicity, these instructions assume local state.
```

File: customer-managed/aws/README.md [hunk 6]
```diff
@@ -1,268 +1,115 @@
...
-      "vpc": {
-        "arn": "<vpc_arn from outputs>"
-      }
-    }
-  }
-}
-EOF
-```
+The [variables.tf]() file contains a number of variables that allow you to modify this code to meet your specific needs.
+In some cases they enable you to skip creation of certain resources (e.g. the VPC) or to modify the configuration of a
```
Comment (micheleRP) [inline customer-managed/aws/README.md:28]:
```suggestion
In some cases, they let you skip creation of certain resources (for example, the VPC) or modify the configuration of a
```

File: customer-managed/aws/README.md [hunk 7]
```diff
@@ -1,268 +1,115 @@
...
+resource.
 
-The create network request will return a resource_id, you will need this in the create cluster step next. Example:
+Create a JSON file called `byoc.auto.tfvars.json` inside the terraform directory.
```
Comment (micheleRP) [inline customer-managed/aws/README.md:31]:
```suggestion
Create a JSON file called `byoc.auto.tfvars.json` inside the Terraform directory.
```

File: customer-managed/aws/README.md [hunk 8]
```diff
@@ -1,268 +1,115 @@
...
+  "zones": [],
+  "enable_private_link": true|false,
+  "create_rpk_user": true|false,
+  "force_destroy_cloud_storage": true|false
 }
-EOF
 ```
 
-The create cluster request will return a resource_id, you will need this in the next step. Example:
+| Variable Name               | Explanation                                                                                                                                                                                                                                                                                                                                                                                                     | Default                                                                                   | Is Required                                                             |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:55]:
```suggestion
| Variable Name               | Description                                                                                                                                                                                                                                                                                                                                                                                                     | Default                                                                                   | Required                                                             |
```

File: customer-managed/aws/README.md [hunk 9]
```diff
@@ -1,268 +1,115 @@
...
+| aws_account_id              | AWS account id in which to create the resources, if not already authenticated via the methods described [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration).                                                                                                                                                                                             | none                                                                                      | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:57]:
```suggestion
| aws_account_id              | AWS account ID in which to create the resources, if not already authenticated via the methods described [here](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration).                                                                                                                                                                                             | none                                                                                      | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 10]
```diff
@@ -1,268 +1,115 @@
...
+| region                      | [AWS region](https://docs.redpanda.com/redpanda-cloud/reference/tiers/byoc-tiers/#tabs-1-amazon-web-services-aws) in which to create the resources                                                                                                                                                                                                                                                              | none                                                                                      | yes                                                                     |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:58]:
```suggestion
| region                      | [AWS region](https://docs.redpanda.com/redpanda-cloud/reference/tiers/byoc-tiers/#tabs-1-amazon-web-services-aws) in which to create the resources.                                                                                                                                                                                                                                                              | none                                                                                      | yes                                                                     |
```

File: customer-managed/aws/README.md [hunk 11]
```diff
@@ -1,268 +1,115 @@
...
+| condition_tags              | Key/value pairs for tags that will be included as conditions on any IAM policies where supported. e.g. You can include `cluster:staging-east` as a condition tag and permissions will be restricted to only those resources that include the tag `cluster:staging-east`. Be aware that any tag included here must also be provided during cluster creation in the `cloud_provider_tags` field of the Cloud API. | redpanda-managed: true                                                                    | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:60]:
```suggestion
| condition_tags              | Key/value pairs for tags that will be included as conditions on any IAM policies where supported. For example, you can include `cluster:staging-east` as a condition tag, and permissions will be restricted to only those resources that include the tag `cluster:staging-east`. Note that any tag included here must also be provided during cluster creation in the `cloud_provider_tags` field of the Cloud API. | redpanda-managed: true                                                                    | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 12]
```diff
@@ -1,268 +1,115 @@
...
+| ignore_tags                 | Tags which if present on these resources in the cloud, but not in this code, will be ignored.                                                                                                                                                                                                                                                                                                                   | none                                                                                      | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:62]:
```suggestion
| ignore_tags                 | Tags which, if present on these resources in the cloud but not in this code, will be ignored.                                                                                                                                                                                                                                                                                                                   | none                                                                                      | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 13]
```diff
@@ -1,268 +1,115 @@
...
+| vpc_id                      | ID of the VPC if created external to this Terraform code. When provided this code will skip creation of the VPC.                                                                                                                                                                                                                                                                                                | none                                                                                      | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:63]:
```suggestion
| vpc_id                      | ID of the VPC, if created external to this Terraform code. When provided, this code skips creation of the VPC.                                                                                                                                                                                                                                                                                                | none                                                                                      | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 14]
```diff
@@ -1,268 +1,115 @@
...
+| private_subnet_ids          | IDs of the private subnets associated with the provided `vpc_id` if created external to this Terraform code. When provided this code will skip creation of the private subnets.                                                                                                                                                                                                                                 | none                                                                                      | Either `private_subnet_ids` or `private_subnet_cidrs` must be provided. |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:64]:
```suggestion
| private_subnet_ids          | IDs of the private subnets associated with the provided `vpc_id`, if created external to this Terraform code. When provided, this code skips creation of the private subnets.                                                                                                                                                                                                                                 | none                                                                                      | Either `private_subnet_ids` or `private_subnet_cidrs` must be provided. |
```

File: customer-managed/aws/README.md [hunk 15]
```diff
@@ -1,268 +1,115 @@
...
+| private_subnet_cidrs        | One subnet will be created per CIDR block in this list. If this list is blank then no subnets will be created, but `private_subnet_ids` will then be required.                                                                                                                                                                                                                                                  | "10.0.0.0/24", "10.0.2.0/24", "10.0.4.0/24", "10.0.6.0/24", "10.0.8.0/24", "10.0.10.0/24" | Either `private_subnet_ids` or `private_subnet_cidrs` must be provided. |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:65]:
```suggestion
| private_subnet_cidrs        | One subnet is created per CIDR block in this list. If this list is blank, then no subnets are created and `private_subnet_ids` is required.                                                                                                                                                                                                                                                  | "10.0.0.0/24", "10.0.2.0/24", "10.0.4.0/24", "10.0.6.0/24", "10.0.8.0/24", "10.0.10.0/24" | Either `private_subnet_ids` or `private_subnet_cidrs` must be provided. |
```

File: customer-managed/aws/README.md [hunk 16]
```diff
@@ -1,268 +1,115 @@
...
+| public_subnet_cidrs         | One subnet will be created per CIDR block in this list and associated to the main route table. If no CIDRs are provided no subnets will be created. It is not common to provide these.                                                                                                                                                                                                                          | "10.0.1.0/24", "10.0.3.0/24", "10.0.5.0/24", "10.0.7.0/24", "10.0.9.0/24", "10.0.11.0/24" | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:66]:
```suggestion
| public_subnet_cidrs         | One subnet is created per CIDR block in this list and associated to the main route table. If no CIDRs are provided, no subnets are created. It is not common to provide these.                                                                                                                                                                                                                          | "10.0.1.0/24", "10.0.3.0/24", "10.0.5.0/24", "10.0.7.0/24", "10.0.9.0/24", "10.0.11.0/24" | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 17]
```diff
@@ -1,268 +1,115 @@
...
+| zones                       | When creating subnets they will be created in these [AWS availability zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#az-ids).                                                                                                                                                                                                                                 | none                                                                                      | Only when `private_subnet_cidrs` or `public_subnet_cidrs` is provided.  |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:67]:
```suggestion
| zones                       | Subnets are created in these [AWS availability zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#az-ids).                                                                                                                                                                                                                                 | none                                                                                      | Only when `private_subnet_cidrs` or `public_subnet_cidrs` is provided.  |
```

File: customer-managed/aws/README.md [hunk 18]
```diff
@@ -1,268 +1,115 @@
...
+| enable_private_link         | When true additional permissions that are required by [AWS Private Link](https://docs.redpanda.com/redpanda-cloud/networking/aws-privatelink/) will be granted.                                                                                                                                                                                                                                                 | false                                                                                     | no                                                                      |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:68]:
```suggestion
| enable_private_link         | When true, additional permissions that are required by [AWS Private Link](https://docs.redpanda.com/redpanda-cloud/networking/aws-privatelink/) will be granted.                                                                                                                                                                                                                                                 | false                                                                                     | no                                                                      |
```

File: customer-managed/aws/README.md [hunk 19]
```diff
@@ -1,268 +1,115 @@
...
+| create_rpk_user             | When true IAM policies reflecting the permissions required by the user running RPK will be created. This is not commonly needed.                                                                                                                                                                                                                                                                                | false                                                                                     | yes                                                                     |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:69]:
```suggestion
| create_rpk_user             | When true, IAM policies reflecting the permissions required by the user running RPK will be created. This is not commonly needed.                                                                                                                                                                                                                                                                                | false                                                                                     | yes                                                                     |
```

File: customer-managed/aws/README.md [hunk 20]
```diff
@@ -1,268 +1,115 @@
...
+| force_destroy_cloud_storage | When true the cloud storage bucket will be destroyed when this Terraform is destroy. Not recommended for production.                                                                                                                                                                                                                                                                                            | false                                                                                     | yes                                                                     |
```
Comment (micheleRP) [inline customer-managed/aws/README.md:70]:
```suggestion
| force_destroy_cloud_storage | When true, the cloud storage bucket is destroyed when this Terraform is destroyed. Not recommended for production.                                                                                                                                                                                                                                                                                            | false                                                                                     | yes                                                                     |
```

File: customer-managed/aws/README.md [hunk 21]
```diff
@@ -1,268 +1,115 @@
...
+
+## Initialize the Terraform
+
+Initializes the working directory containing Terraform configuration files.
```
Comment (micheleRP) [inline customer-managed/aws/README.md:74]:
```suggestion
Initialize the working directory containing Terraform configuration files.
```

File: customer-managed/aws/README.md [hunk 22]
```diff
@@ -1,268 +1,115 @@
...
-# Wait for cluster creation to complete
-
-At this point the agent VM is running and will handle the remaining setup steps. This can take up to 45 minutes. When it 
-is complete the cluster status should be updated to ‘Running’. If the cluster status is still ‘Creating’ after that 
-time, reach out to Redpanda support to investigate.
-
-You can check the cluster status via the API or using the website.
-
-Example using returned `operation_id`:
+The output of `terraform apply` should display a number of output values.
```
Comment (micheleRP) [inline customer-managed/aws/README.md:88]:
```suggestion
The output of `terraform apply` should display a number of output values. For example:
```

File: customer-managed/aws/README.md [hunk 23]
```diff
@@ -1,268 +1,115 @@
...
+These values can also be retrieved at any time by running `terraform output`.
 
-```shell
-curl -X GET "https://api.redpanda.com/v1beta2/clusters/<cluster_id>" \
-  -H "accept: application/json"\
-  -H "content-type: application/json" \
-  -H "authorization: Bearer $PUBLIC_API_TOKEN"
-```
+Note these values. They are needed for the next steps. To continue with cluster creation please see 
+[Configure a Customer-Managed VPC on AWS](https://deploy-preview-12--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/byoc/vpc-byo-aws/).
```
Comment (micheleRP) [inline customer-managed/aws/README.md:115]:
```suggestion
Note these values. They are needed for the next steps. To continue with cluster creation, see 
[Configure a Customer-Managed VPC on AWS](https://deploy-preview-12--rp-cloud.netlify.app/redpanda-cloud/get-started/cluster-types/byoc/vpc-byo-aws/). 
```

**Review by micheleRP — APPROVED** [review]
lots of little suggestions, but looks great!

**Review by paulzhang97 — APPROVED** [review]

**Pull request merged**